portfolio/src/main/webapp/scripts/results.js

```diff
@@ -1,20 +1,37 @@
 const baseUri = 'https://api.spotify.com/v1';
 const accessTokenKey = 'access_token';
 
-// Hardcoded results.
 var songNum = 1;
-var seedArtists = '7Ln80lUS6He07XvHI8qqHH'; 
-var seedGenres = 'alternative';
-var seedTracks = '5ruzrDWcT0vuJIOMW7gMnW';
 
 // Gets result based on quiz results.
 function getResult() {
-    // Need to change options here to match new quiz and get values from URL query params.
+    // Get values from URL query params.
+    let seedGenres = null;
+    let targetValence = null;
+    let targetPopularity = null;
+    let targetTempo = null;
+    let targetAcousticness = null;
+    const queryString = window.location.search;
+    if(queryString.length > 0)
+    {
+        const urlParams = new URLSearchParams(queryString);
+        seedGenres = urlParams.get('genre');
+        targetValence = urlParams.get('valence');
+        targetPopularity = urlParams.get('popularity');
+        targetTempo = urlParams.get('bpm');
+        targetAcousticness = urlParams.get('acousticness');
+        targetValence = targetValence/100;
```

Comment:
Would recommend doing the divide by 100 inline with the urlParams.get.
For example: `targetValence = urlParams.get('valence') / 100;`

Otherwise just glancing at this, it almost looks like we're trying to set 8 different parameters instead of 5.